lib/blockscore/base.rb

```diff
@@ -5,6 +5,7 @@ class Base
     extend Connection
 
     ABSTRACT_WARNING = 'Base is an abstract class, not an API resource'.freeze
+    API_URL = 'https://api.blockscore.com/'.freeze
```

Comment:
Maybe you should make this an instance of `Addressable::URI`, this is like `Pathname` for file paths a much better abstraction for the concept of URI than a string.